src/Integration.Vsix/Analysis/AnalysisConfigMonitor.cs

```diff
@@ -67,7 +67,7 @@ private void OnUserSettingsChanged(object sender, EventArgs e)
             if (activeSolutionBoundTracker.CurrentConfiguration.Mode == NewConnectedMode.SonarLintMode.Standalone)
             {
                 logger.WriteLine(AnalysisStrings.ConfigMonitor_UserSettingsChanged);
-                analysisRequester.RequestAnalysis();
+                analysisRequester.RequestAnalysis(null, null);
```

Comment:
I think it's worth creating an overload that doesn't accept parameters. Both of them are optional, so passing nulls seems a bit messy.

Reply:
As discussed offline, I've added an extension method: 
```
public static void RequestAnalysis(this IAnalysisRequester analysisRequester, params string[] filePaths)
```